webapp/src/main/resources/public/js/ict/IctModalContainer.js

```diff
@@ -43,13 +50,24 @@ class IctModalContainer extends React.Component {
         window.open(this.state.mojitoBaseUrl + "workbench?" + query);
     }
 
+    openActionInNewWindow(url) {
+        var query = queryString.stringify({
+            'repoName': this.state.selectedTextUnit.repositoryName,
+            'locales': this.state.selectedTextUnit.locale,
+            'searchText': this.state.selectedTextUnit.textUnitName
+        });
+        window.open(url + '&' + query);
```

Comment:
'&' should be '?' which is delimiter between the base url and the query string, no?

Reply:
The url I was using actually already had an initial query parameter in it. I'll think about updating it.